bson/src/test/unit/org/bson/DocumentTest.java

```diff
@@ -108,6 +111,53 @@ public void shouldGetWithDefaultValue() {
         assertEquals("bar", z2);
     }
 
+    @Test
+    public void shouldGetListFromDocument() {
```

Comment:
Note: All the other `Document.get` methods are tested in the `DocumentSpecification.groovy` spock test. These should be moved there.

Reply:
I used the GoTo / Test option in Intellij for the `get` method, and the first option was the JUnit tests, which I selected, thinking that was the default. I forgot that we have spock tests for the unit tests.

Reply:
No worries - we have *a lot* of Spock tests. Spock tests can be great as they aren't as verbose as JUnit tests. But there are a lot of groovy-isms to learn and occasionally some gotchas. So might be fun to migrate over. (Beware codenarc our automated groovy style check tests is particularly thorough)